index.html

```diff
@@ -55,29 +56,30 @@
     <div class="container-fluid">
       <div class="row masthead">
         <div class="row title-banner">
-          <h1 class="text-center">Theta Gamma Chapter</h2>
-          <h2 class="text-center">University of Michigan</h3>
+          <h1 class="text-center">Theta Gamma Chapter</h1>
+          <h2 class="text-center">University of Michigan</h2>
         </div>
-
-
         <!-- quote -->
         <div id="quote" class="row">
           <div class="col-xs-6 col-xs-offset-3 quote-block">
```

Comment:
better way to position quote block. allows different width for mobile etc.  using `margin: auto`